frontend/packages/topology/integration-tests/support/step-definitions/topology/application-groupings.ts

```diff
@@ -62,3 +66,29 @@ Then(
     topologyPage.verifyApplicationGroupingSidepane();
   },
 );
+
+Then('user hovers on Add to Application from action menu', () => {
```

Comment:
make sure the step definitions match the feature file (e.g. this is supposed to be a `When` step)